experiments/functional/zfs-LocalPV/zfs-controller-HA/run_litmus_test.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: zfs-controller-high-avail
```

Comment:
@w3aman  provide the full name don't cut the name `availability`

Reply:
Changed !